server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+let app = express();
+const parser = require('body-parser');
+const database = require('./db/index.js');
+
+
+app.use(express.static(__dirname + '/../client/dist'));
+app.use(parser.json());
+app.use(parser.urlencoded({ extended: true }))
+let port = process.env.PORT || 1128;
+app.listen(port, function () {
+  console.log(`listening on port ${port}`);
+});
+
+
+app.get('/', (req, res) => {
+  res.send(200);
```

Comment:
```suggestion
  res.sendStatus(200);
```
per error in server console: `express deprecated res.send(status): Use res.sendStatus(status)`